app/models/company_value.rb

```diff
@@ -1,3 +1,5 @@
 class CompanyValue < ApplicationRecord
+  has_many :kudos, dependent: nil
```

Comment:
It may be not the best option, because we will end up with the kudos pointing to CompanyValues, that are not existing, which most likely blow up some views
```suggestion
  has_many :kudos, dependent: destroy
```

Reply:
I considered it but now nobody can remove CompanyValues therefore I set dependence for nil. But I agree that was not the best option. I corrected it.